resources/lang/nld.ini

```diff
@@ -17,13 +17,13 @@ error.limitexceeded = "U probeert te veel blokken tegelijk te bewerken. Verklein
 error.notarget = "Geen doelblok gevonden. Vergroot gereedschapsbereik met //setrange indien nodig"
 error.noselection = "Geen selectie gevonden - selecteer eerst een gebied"
 error.selectioninvalid = "De selectie is niet geldig! Controleer of alle posities zijn geselecteerd!"
-error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {% 0} te gebruiken"
+error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {%0} te gebruiken"
 error.noclipboard = "Geen klembord gevonden - maak eerst een klembord"
 warning.differentlevel = "[WAARSCHUWING] Je bewerkt een wereld waar je momenteel niet in zit!"
```

Comment:
Oh btw this changed to .differentworld recently, mind patching?

Reply:
I've changed .nosession to .differentworld, is that what you intended?

Reply:
should've been more specific, `differentlevel` -> `differentworld`